eldritch/events.py

```diff
@@ -1301,6 +1313,17 @@ def __init__(self, character, location_name):
     self.done = False
 
   def resolve(self, state):
+    if (isinstance(self.location_name, str)
+            and getattr(state.places[self.location_name], "closed_until", None) is not None):
+      destinations = {place.name for place in state.places[
```

Comment:
Shouldn't you also check for the other places being closed and only offer the player open locations?

Reply:
I've now added a test to ensure they can't choose a closed location.